### PR TITLE
Manual block production in standalone

### PR DESCRIPTION
**Pull request by savudani8**
This PR replaces the consensus pallet in the standalone binary from `aura` to `manual-seal`. The standalone mode can only serve the purpose of testing henceforth. 

Not sure if this requires bumping the minor or the major SemVer version?

### CODE REVIEW

**Review by gregdhill — CHANGES_REQUESTED** [review]
@savudani8 can you cleanup the aura & grandpa dependencies? I think we can also remove some of these from the runtime crate too right?

File: standalone/src/service.rs [hunk 1]
```diff
@@ -184,131 +146,72 @@ pub fn new_full(mut config: Configuration) -> Result<(TaskManager, RpcHandlers),
...
+        config.prometheus_registry(),
+        None,
+    );
+
+    // Channel for the rpc handler to communicate with the authorship task.
+    let (mut command_sink, commands_stream) = mpsc::channel(10);
+
+    let rpc_sink = command_sink.clone();
+
+    let fut = transaction_pool.import_notification_stream().map(
```
Comment (sander2) [inline standalone/src/service.rs:163]: Maybe I'm missing something, but it doesn't look like this map is doing anything - `map` is lazy and doesn't do any work unless you do something with the result

**Comment by gregdhill** [comment]
See also: https://github.com/paritytech/substrate/issues/10239

**Comment by sander2** [comment]
I tested without docker again, but finalization still isn't working properly. I did some debugging and narrowed down the problem to the `author_extrinsicUpdate` message not being sent to the client with the finalized status, see https://github.com/paritytech/substrate/issues/10279

**Pull request closed without merging**